rledger.yaml

```diff
@@ -0,0 +1,90 @@
+owner: test-user
```

Comment:
was this accidentally committed?

Reply:
Not really. I contemplated adding it a few times. It gives a pretty solid example of a use case. Actually ledger offers one as well. If you think we should remove it then let me know I Ill pull it from the PR.

Reply:
oh, that is a good idea. Yeah, I think we should include it. Maybe we could it put it in its own subdir (ie `/example`) and reference it in the `README` somewhere?